### DISCUSSION TITLE
For a list literal, mouse over an element to show its index

### DISCUSSION THREAD

**Discussion by Daniel-Chin** [microsoft/pylance-release#5112] [Enhancement]
In the Python language mode, mouse over an element in a list literal to know its current index number. This can be an extension.

**Comment by rchiodo** [comment]
Sorry can you give more information on what you're hoping for?

Something like this?

```python
#             | hover here and get index 3?
x = [1, 2, 3, 4, 5, 6]
```

**Reply by Daniel-Chin** [reply]
Yes! Exactly.

**Comment by rchiodo** [comment]
Converting to a discussion for up votes.
